### ISSUE TITLE
Changed event

### ISSUE THREAD

**Issue by christianecker** [AndyMeps/ng2-dropdown-multiselect#6]
Is there an event I can use, which is called on model changes?

Best regards

Christian

**Comment by AndyMeps** [comment]
Hi Christianecker,

I've not tried it but something like this should work:

 ```javascript
<dropdown-multiselect 
    [ngModel]="dropdownModel" 
    (ngModelChange)="onChange($event)">
</dropdown-multiselect>
``` 

Where `onChange()` is a function in your calling component and `dropdownComponent` is the model property in your calling component.

This is part of the `@angular/forms` NgModel class, see more details [here](https://angular.io/docs/ts/latest/api/forms/index/NgModel-directive.html) if necessary.

I'll leave this open for the time being in case this isn't what you were after.

Thanks for using my library and I hope it helps! :)

**Comment by christianecker** [comment]
Hi Andy,

thanks for the fast reply. unfortunately you suggested solution is not working for me - the onChange method is not invoked :-/

fyi my test method looks like this:

public onChange(event): void {
    console.log(event);
  }

**Comment by AndyMeps** [comment]
Hi christianecker,

I've just published v0.3.3 which should now call the `ngModelChange` event handler when toggling a dropdown item or selecting "Check All" / "Uncheck All". I hope this fits your requirements? It will return the entire state of the model post-change.

Thanks for raising the issue and I hope my library helps! :)

Andy
